pkg/proxy/winkernel/proxier.go

```diff
@@ -182,12 +182,14 @@ func (ep *endpointsInfo) Cleanup() {
 	// Never delete a Local Endpoint. Local Endpoints are already created by other entities.
 	// Remove only remote endpoints created by this service
 	if ep.refCount <= 0 && !ep.isLocal {
```

Comment:
so ep.refCount doesn't catch such cases?

Reply:
Can't, two different ep are  created in such case, all of them have refCount 1.
Deleting anyone of them will trigger deletion of hnsEdnpoint.